css/backoffice/blocks-integrations/_all.scss

```
@@ -22,4 +22,5 @@
 @import "medallion-with-blocklist";
 @import "field-badge-within-datatable";
 @import "jquery-blockui-within-dialog";
-@import "jquery-blockui-within-datatable";
\ No newline at end of file
+@import "jquery-blockui-within-datatable";
+@import "summary-card-within-tippy";
```

```suggestion
@import "object-summary-within-tippy";
```

See comment below.
Also move this under the object-details partial.

---

Done in [13320b8](https://github.com/Combodo/iTop/pull/476/commits/13320b87bbb5c3492b82ba76ba92d14f2dfac4be)